azkaban-common/src/test/java/azkaban/container/models/AzKubernetesV1PodBuilderTest.java

```diff
@@ -0,0 +1,39 @@
+package azkaban.container.models;
+
+import azkaban.utils.TestUtils;
+import com.google.common.collect.ImmutableMap;
+import io.kubernetes.client.openapi.models.V1Pod;
+import io.kubernetes.client.util.Yaml;
+import java.io.IOException;
+import java.util.Optional;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class AzKubernetesV1PodBuilderTest {
+    @Test
+    public void testPodSpec() throws IOException {
+        String podName = "az-example";
+        String podNameSpace = "az-team";
+        String clusterName = "mycluster";
+        ImmutableMap<String, String> labels = ImmutableMap.of("lkey1", "lvalue1");
+        ImmutableMap<String, String> annotations = ImmutableMap.of("akey1", "aval1");
+        String jobTypeName = "spark";
+        String jobTypeImage = "path/spark-jobtype:0.0.5";
+        String imagePolicy = "IfNotPresent";
```

Comment:
Can you please write enum for this policy?

Reply:
Sure